app/components-react/windows/go-live/GameSelector.tsx

```diff
@@ -74,6 +74,7 @@ export default function GameSelector(p: TProps) {
       imageSize={TwitchService.gameImageSize}
       loading={isSearching}
       notFoundContent={isSearching ? $t('Searching...') : $t('No matching game(s) found.')}
+      allowClear
```

Comment:
I we want to keep it the same for Twitch we should use `allowClear={!isTwitch}`

Reply:
do you think we should? the field is required for Twitch and we have a warning if it's cleared saying the field is required. IMO this is a better experience (I personally prefer to clear the field before searching)